api_tests/tests/reasonable_adjustments/test_asid.py

```diff
@@ -15,23 +17,8 @@ def test_valid_asid(self, get_token):
         expected_asid = '200000001115'
 
         # When
-        self.send_a_request()
```

Comment:
I think it'd be better to leave this method as it it is. The reason behind this `send_a_request` method is that the test has nothing to do with which endpoint we're calling. Extracting this into a separate method makes the test more focus on what matters.